docs/deprecations.md

```diff
@@ -13,7 +13,7 @@ weight: 107
 ## Introduction
 
 This doc provides a list of features in Tekton Pipelines that are
-being deprecated.
+being deprecated and are already deprecated.
 
 ## Deprecation Table
```

Comment:
suggest adding a comment that these features are deprecated but have not yet been removed, and rename this to "deprecated features"

Reply:
Thanks Lee! That makes it much clearer